bumble/host.py

```diff
@@ -1068,6 +1060,8 @@ def on_hci_connection_complete_event(
                 None,
                 None,
                 None,
```

Comment:
The last 3 parameters should be `0, 0, 0` instead of `None, None, None` since they are not declared as `Optional`.